test/sim/InitializeTracks.test.cc

```diff
@@ -0,0 +1,160 @@
+//---------------------------------*-CUDA-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file InitializeTracks.test.cc
+//---------------------------------------------------------------------------//
+#include "InitializeTracks.test.hh"
+
+#include "physics/base/SecondaryAllocatorStore.hh"
+#include "gtest/Main.hh"
+#include "gtest/Test.hh"
+#include "sim/InitializeTracks.hh"
```

Comment:
To guard against missing includes in the source file, it's best to put the header for the class being tested at the very top of the file.